api/datastore/test_data/output_24_to_latest.json

```diff
@@ -62,6 +62,9 @@
           "UseLoadBalancer": false,
           "UseServerMetrics": false
         },
+        "Flags": {
+          "IsServerMetricsDetected": false
+        },
```

Comment:
Maybe I'm confused, but shouldn't this be "true" if the migration is working properly?

Reply:
yeah, you are right. Let me check if the migration is running on the this test DB.

Reply:
(Prabhat found that our test endpoint is type docker not kubernetes so this is the correct result)